HumanControls/Page.cpp

```diff
@@ -0,0 +1,70 @@
+#include "Page.h"
+
+Page::Page()
+{
+    this->isActive = false;
+    this->modifyValue = 0;
+    increment = 0;
+    this->min = 0;
+    this->max = 0;
+}
+
+Page::Page(int increment, int min, int max, bool isActive)
+{
+    this->isActive = isActive;
+    this->modifyValue = min;
+    this->min = min;
+    this->max = max;
+    this->increment = increment;
+}
+
+void Page::cleanUp(DisplayController &display)
+{
+    display.clear();
+}
+void Page::paint(DisplayController &display, bool isActivated)
+{
+    display.clear();
+    display.printRegion(2, 0, "On a page");
+}
+
+void Page::clockwise()
+{
```

Comment:
This code will most likely need to be page-specific as the JSON code is added. But this is ok for now.

Reply:
I know, figured I would leave it like this until the Json stuff was ready, and I have a better idea on how I want to implement it.